rest/types/endpoint.go

```diff
@@ -2,3 +2,8 @@ package types
 
 // EndpointPrefix is a type specifying the endpoint on which the resource exists.
 type EndpointPrefix string
+
+const (
+	// ExtendedEndpoint - All endpoints added through additional servers which are part of the core API.
+	ExtendedEndpoint EndpointPrefix = "1.0"
```

Comment:
This doesn't need to exist anymore since we allow projects to specify their own prefixes for the core API, as long as they don't conflict with one of the existing ones.

Reply:
Fair point, removed.